subsys/logging/Kconfig

```diff
@@ -185,6 +185,7 @@ config LOG_MAX_LEVEL
 
 config LOG_PRINTK
 	bool "Enable processing of printk messages."
+	default y
```

Comment:
This will ugly clash with shell when boot banner is enabled :/.

Reply:
it shouldn't. That has been fixed with #10510